circle.yml

```diff
@@ -20,3 +20,5 @@ deployment:
     tag: /[0-9]+(\.[0-9]+)*(-.+)?/
     commands:
       - curl -sLu "$HTTP_AUTH:" https://raw.githubusercontent.com/segmentio/scripts/master/circle/docker/build.sh | sh
+      - npm publish --access=restricted
```

Comment:
the fact that npm doesn't allow you to publish a scoped package unless you remove "private" makes me nervous. Added the explicit check to be 100% sure